src/UI/Component/Toast/Factory.php

```diff
@@ -40,4 +38,31 @@ interface Factory
      * @return  \ILIAS\UI\Component\Toast\Toast
      */
     public function standard($title, Icon $icon) : Toast;
+
+    /**
+     * ---
+     * description:
+     *   purpose:
+     *     Toasts Containers displays Toasts in the top right corner of the ILIAS page content.
+     *     It has no self-interested visual purpose.
+     *   composition:
+     *     Toast Containers contain a group of toasts.
+     *   effect:
+     *     The container will be displayed overlaying the main content.
+     *     If the container is empty it will nto be displayed.
+     * rules:
+     *   style:
+     *     1: The Toast Container SHOULD be limited in space so it does not cover all of the pages content, no matter
+     *        the size of its own content.
+     *     2: An empty Toast Container SHOULD have no effect on the visible page.
+     *   responsiveness:
+     *     1: The Toast Container MAY be hidden on devices with a low screen size.
```

Comment:
I don't think so. But we certainly will need some strategy to deal with the thingy on small screens. Please just remove this here. I know we won't solve this now, please add the problem to our ROADMAP.md.